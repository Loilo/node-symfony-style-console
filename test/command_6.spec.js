

const wrapper = require('./wrapper')

test('06 Ensure has proper blank line after text block when using a block like with SymfonyStyle::success', wrapper(io => {
    io.listing([
        'Lorem ipsum dolor sit amet',
        'consectetur adipiscing elit',
    ])
    io.success('Lorem ipsum dolor sit amet')
}))
