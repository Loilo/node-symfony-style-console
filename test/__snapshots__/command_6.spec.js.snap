// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`06 Ensure has proper blank line after text block when using a block like with SymfonyStyle::success 1`] = `
"
 * Lorem ipsum dolor sit amet
 * consectetur adipiscing elit

 [OK] Lorem ipsum dolor sit amet                                                

"
`;
